### ISSUE TITLE
default meta description value

### ISSUE THREAD

**Issue by onno-timmerman** [wagtail/wagtail#9656]
When you start fresh project (wagtail 4.1.1) then you get some nice default file. In your templates you will find base.html and will see that the title of the page is already filled in to the common pattern you will use for wagtail
But the default meta description is **empty**. You can have several meta description strategies but in wagtail we have a dedicated field. 
Maybe this should be in the default markup then

```
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <title>
            {% block title %}
            {% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.title }}{% endif %}
            {% endblock %}
            {% block title_suffix %}
            {% wagtail_site as current_site %}
            {% if current_site and current_site.site_name %}- {{ current_site.site_name }}{% endif %}
            {% endblock %}
        </title>
        <meta name="description" content="" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

to

`<meta name="description" content="{{ page.search_description }}" />`

**Comment by lb-** [comment]
Good idea. Maybe we should have some basic logic to check the description exists (I'm not sure on the rules with meta tags with empty values though - maybe it's ok).

Here are some links to examples from other projects. 

https://github.com/wagtail/guide/blob/main/apps/core/templates/base.html#L16

https://github.com/wagtail/wagtail.org/blob/main/wagtailio/templates/base.html#L8

I'll flag as a good first issue.

**Comment by AnishaSingh0118** [comment]
Hello @onno-timmerman @lb- currently in the description it is showing empty so what exactly should be there could you provide some ideas/examples so that I could work on this issue.(being the good first issue).

**Comment by onno-timmerman** [comment]
Either you place page.search_description 
<meta name="description" content="{{ page.search_description }}" />

Or maybe something basic function you can override. So a dev can say something like

def seo_description(self):
   return self.search_description

That way you can put any some logic in the seo_description. My guess is if you know howto then you can do that easy yourself. Although it always nice to have always same methods over your projects and makes a dev think about the options you have. Example sometimes you want to take the first words of an intro if the search_description is empty.

**Comment by AnishaSingh0118** [comment]
> Either you place page.search_description
> 
> Or maybe something basic function you can override. So a dev can say something like
> 
> def seo_description(self): return self.search_description
> 
> That way you can put any some logic in the seo_description. My guess is if you know howto then you can do that easy yourself. Although it always nice to have always same methods over your projects and makes a dev think about the options you have. Example sometimes you want to take the first words of an intro if the search_description is empty.

Ok, thanks for your help and support.

**Comment by AnishaSingh0118** [comment]
@onno-timmerman .. Could you suggest which one I am trying to fix is appropriate for this particular issue so that I can have a better idea towards it.

1)
![IMG_20221117_171329](https://user-images.githubusercontent.com/76876709/202447850-6f26334b-dfe9-46ad-aba6-3db916539f74.jpg)

2)
![IMG_20221117_171358](https://user-images.githubusercontent.com/76876709/202447881-3167acc2-7840-4f0b-80cc-da63c111a5c2.jpg)

**Comment by onno-timmerman** [comment]
You can fill this in but in the default template that Wagtail deliver base.html the tag is not filled in. It would be nice that its by default there just like the title meta tag

 <meta name="description" content="" />

**Comment by salty-ivy** [comment]
> @onno-timmerman .. Could you suggest which one I am trying to fix is appropriate for this particular issue so that I can have a better idea towards it.
> 
> 
> ![IMG_20221117_171329](https://user-images.githubusercontent.com/76876709/202447850-6f26334b-dfe9-46ad-aba6-3db916539f74.jpg)
> 
> 
> ![IMG_20221117_171358](https://user-images.githubusercontent.com/76876709/202447881-3167acc2-7840-4f0b-80cc-da63c111a5c2.jpg)

@AnishaSingh0118 it is very much possible to fix it through manual input,
the issue is all about default behaviour and its customization ,

in case if you are stuck , let me know you should first open a draft PR i will help you out through the commits

**Comment by AnishaSingh0118** [comment]
> > @onno-timmerman .. Could you suggest which one I am trying to fix is appropriate for this particular issue so that I can have a better idea towards it.
> > ![IMG_20221117_171329](https://user-images.githubusercontent.com/76876709/202447850-6f26334b-dfe9-46ad-aba6-3db916539f74.jpg)
> > ![IMG_20221117_171358](https://user-images.githubusercontent.com/76876709/202447881-3167acc2-7840-4f0b-80cc-da63c111a5c2.jpg)
> 
> @AnishaSingh0118 it is very much possible to fix it through manual input, the issue is all about default behaviour and its customization ,
> 
> in case if you are stuck , let me know you, should first open a draft PR i will help you out through the commits

Ok, thanks for your support. I making a draft.

**Comment by salty-ivy** [comment]
@onno-timmerman  should i take this one up?

**Comment by salty-ivy** [comment]
@lb-  i have made the PR https://github.com/wagtail/wagtail/pull/9764

**Comment by lb-** [comment]
Awesome @salty-ivy  - I have marked it as needing a review

**Comment by gasman** [comment]
Fixed in #9764
